### PR TITLE
Adding Python example

### PR DESCRIPTION
**Pull request by tpmccallum**
Ready for approval and merge.
Tested and is working.

### CODE REVIEW

**Comment by tpmccallum** [comment]
`AssertionError: Error::RuntimeError("llm feature is not enabled")` when using `curl -X POST --data '{"sentence":"Everything is awesome!"}' https://sentiment-analysis-zxrl7aq1.fermyon.app` to test the deployed app.

File: sentiment-analysis-py/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Clone this ai-examples repository:
+
+```bash
+$ git clone https://github.com/fermyon/ai-examples.git
+```
+
+# Build the Application
+
+```bash
+$ cd cd ai-examples/sentiment-analysis-py
```
Comment (rajatjindal) [inline sentiment-analysis-py/README.md:35]: double cd here

File: sentiment-analysis-py/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+$ curl -X POST --data '{"sentence":"Everything is awesome!"}' https://sentiment-analysis-abc-xyz.fermyon.app/
+```
+
+## Result
+
+The result is returned to you as a JSON String
+
+```json
+{
+    "sentence": "Bot: positive"
```
Comment (rajatjindal) [inline sentiment-analysis-py/README.md:61]: this output looks different then our other examples.

File: sentiment-analysis-py/README.md (hunk 1 above)
Comment (tpmccallum) [inline sentiment-analysis-py/README.md:35]:
```suggestion
$ cd ai-examples/sentiment-analysis-py
```

File: sentiment-analysis-py/README.md (hunk 2 above)
Comment (tpmccallum) [reply sentiment-analysis-py/README.md:61]: Yes, this example is just doing AI inferencing via curl (does not use the KV store or static file server etc.).
Just a Serverless AI example only; on purpose :)

File: sentiment-analysis-py/app.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Bot: negative
+[/INST]
+
+User: """
+
+def handle_request(request):
+    request_body=json.loads(request.body)
+    sentence=request_body["sentence"].strip()
+    result=llm_infer("llama2-chat", PROMPT+sentence)
+    response_body=json.dumps({"sentence": result.text})
```
Comment (tpmccallum) [inline sentiment-analysis-py/app.py:27]:
```suggestion
    response_body=json.dumps({"sentence": re.sub("\\nBot\: ", "", result.text)})
```

File: sentiment-analysis-py/app.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from spin_http import Response
+from spin_llm import llm_infer
+import json
+
```
Comment (tpmccallum) [inline sentiment-analysis-py/app.py:4]:
```suggestion
import re

```

File: sentiment-analysis-py/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
```
Comment (tpmccallum) [inline sentiment-analysis-py/README.md:61]:
```suggestion
    "sentence": "positive"
```

File: sentiment-analysis-py/README.md (hunk 2 above)
Comment (tpmccallum) [reply sentiment-analysis-py/README.md]: Thanks @rajatjindal 
Understand that you meant the actual response text (edited to exclude the `Bot: `. Changes are in the latest code.
```
{"sentence": " positive"}
```

**Pull request merged**